src/twitch.py

```diff
@@ -1,13 +1,16 @@
 import os
 import threading
+import aiocron
 from twitchio.ext import commands
 from leaderboardBot import LeaderBoardBot
 from parseRegion import parseRegion, isRegion
-from channels import channels
 from dotenv import load_dotenv
 
+
 load_dotenv()
 
+channels = {'liihs': 'lii',}
```

Comment:
If I restart the containers, will it have to wait until the cron job runs to join more channels?